src/index.ts

```
@@ -0,0 +1,347 @@
+      generateSitemap(module.baseUrl, pages)
+    );
+    return { name, xml, baseUrl: module.baseUrl };
+  };
+
+  const totalPagesCount = pages.reduce((sum, loc) => sum + loc.pages.length, 0);
+  if (totalPagesCount < PAGINATION_LIMIT)
+    return [
+      getSitemap(
+        `sitemap-${module.name}`,
+        pages.reduce(
+          (allPages, locale) => [...allPages, ...locale.pages],
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)